integrations/tekton/README.md

```diff
@@ -0,0 +1,28 @@
+# Tekton Pipeline Example
+
+This example pipeline allows you to easily run an ORT scan on [Tekton](https://tekton.dev/), some knowledge about Tekton will help to understand this example pipeline.. This example pipeline works best on [Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine).
```

Comment:
- Maybe just "run ORT on"?
- Start a new sentence from "Some knowledge..."?
- s/pipeline../pipeline./
- It is really true that this "works best" on GKE, or has it only been tested on GKE? If indeed the former, could you briefly add why?

Reply:
Also, we should probably explain acronyms once before using them later, like

[Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine) (GKE)